spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/orm/jpa/HibernateProperties.java

```diff
@@ -121,6 +138,10 @@ else if (!result.containsKey(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS)) {
 		}
 	}
 
+	private void applyArchiveScanner(Map<String, Object> result) {
+		result.put(AvailableSettings.SCANNER, this.archiveScanner);
```

Comment:
That's not checking if a user has set the property

Reply:
added checking